### DISCUSSION TITLE
I get only that yellow "update" log message, regardless what i do to example.yaml code.

### DISCUSSION THREAD

**Discussion by Jkirjo** [omerfaruk-aran/esphome_samsung_hvac_bus#182] [General]
Tried hours and hours now. Anything seems not make a change.
I have Atom S3 lite and Atom rs-485 tail, changed rx and tx pins to correct. And my Samsung device is in compatible list. As in header, i only get always that yellow log message "update". Reversed data wires, tried lower baudrate and many other modification, no success.
Sorry my english.
There are many lines in exsample.yaml, which are not allowed to compiler without removing or modify. Impossible to install, when get error. I'll try to add my latest "first run" yaml, if somebody can see, what is wrong. I have tried dozens of different versions, all is not allowed or don't work, if compiler allows it. I have no idea anymore.
```
esphome:
  name: samsung_ac
  friendly_name: Climate

esp32:
  board: esp32-s3-devkitc-1
  framework:
    type: arduino

# Enable logging
logger:
  baud_rate: 0
  level: INFO
  logs:
    component: ERROR # Remove the "Your component takes to long to respond warning"

# Enable Home Assistant API
api:

# Allow Over-The-Air updates
ota:
- platform: esphome

# Allow provisioning Wi-Fi via serial
#improv_serial:   #poistanpa tämänkin vielä, kun ei ole alkuperäisessä ja ei vaan toimi.

wifi:
  networks:
  - ssid: Sauna
    password: xxxx
  - ssid: Plh
    password: xxxx
  - ssid: Kellari
    password: xxxx

# In combination with the `ap` this allows the user
# to provision wifi credentials to the device via WiFi AP.
#captive_portal:

#dashboard_import:
 # package_import_url: github://esphome/firmware/esphome-web/esp32s3.yaml@main
 # import_full_config: true

# Sets up Bluetooth LE (Only on ESP32) to allow the user
# to provision wifi credentials to the device.
esp32_improv:
  authorizer: none

# To have a "next url" for improv serial
web_server:
  port: 80


captive_portal: #Kokeillaas nyt ekana poistaa tää toinen captive kommenttimerkillä.

# Specify pins used by the board to comunicate with RS485 board - in this case M5STACK
uart:
  tx_pin: GPIO5   # Use GPIO26 for ATOM Tail485 Now g5 for S3 lite
  rx_pin: GPIO6   # Use GPIO32 for ATOM Tail485 Now g6 for S3 lite.
  # For older devices it could also be 2400. But you should start with 9600.
  baud_rate: 9600
  parity: EVEN
  debug:
    direction: BOTH
    dummy_receiver: false
    

## SAMSUNG AC CONFIGURATION

# Import custom component from GitHub
external_components:
  - source: github://lanwin/esphome_samsung_ac@main # use @main if you want the latest development (possibly unstable?)
    components: [samsung_ac]
# Configuration of AC component
samsung_ac:
  
  #When enabled (set to true), this option will log the messages associated with undefined codes on the device. This is useful for debugging and identifying any unexpected or unknown codes that the device may receive during operation.
  debug_log_messages_raw: true
  
  #When enabled (set to true), this option logs messages associated with defined codes on the device. This helps in monitoring the behavior of the device by recording the activity related to known, expected codes.
  debug_log_messages: true

  devices:
```

**Comment by lanwin** [comment]
Hi @Jkirjo, if you get only the update log message, its an indication that there is not data incoming. 

First check the https://github.com/lanwin/esphome_samsung_ac?tab=readme-ov-file#troubleshooting section and try the bottom to dump anything coming from serial connection. Please also change the log level to `level: DEBUG` to ensure you dont miss anything.

If there still nothing is in you log, the I would suggest to review the documentation of your ESP and check if your have tx_pin and rx_pin to the right GPIOs.

**Comment by Jkirjo** [comment]
Thanks.
I changed log level to debug, now i also get purple lines...
[15:21:50][C][samsung_ac:040]: Configured devices: 
[15:21:50][C][samsung_ac:060]: Discovered devices:
[15:21:50][C][samsung_ac:061]:   Outdoor: -
[15:21:50][C][samsung_ac:062]:   Indoor:  -
Gpio pins should be correct, document says that this AtomS3 lite tx is gpio5 and rx is gpio6. Atom RS485 tail has no configuration, i think.
RS lines are connected to F1 and F2.

**Comment by Jkirjo** [comment]
Fixed, my own mistake. There was earlier some other error and i tried to switch tx and rx pins for test. And ofcourse forget switch them back correct pins. Now i got addresses to indoor and outdoor units. Outdoor 10.00.00 and indoor 20.00.00
